pkg/lb/lbgrpc/client.go

```diff
@@ -1263,14 +1263,24 @@ func (c *Client) CreateSnapshot(
 			return nil, status.Errorf(codes.Internal,
 				"failed to create snapshot '%s' on LB: %s", name, st.Message())
 		case codes.FailedPrecondition:
-			// most likely source volume is being updated, tell
+			// most likely old snapshot still creating (older Lightbits releases), tell
```

Comment:
most likely old snapshot is still being created (this blocked creating new snapshots in older Lightbits releases)